apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.ts

```diff
@@ -0,0 +1,94 @@
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import path = require('path');
+import debugModule from 'debug';
+import error = require('@prairielearn/error');
+import { v4 as uuidv4 } from 'uuid';
+import { InstructorSharing } from './instructorCourseAdminSharing.html';
+import sqldb = require('@prairielearn/postgres');
+
+const router = Router();
+const debug = debugModule('prairielearn:' + path.basename(__filename, '.js'));
+const sql = sqldb.loadSqlEquiv(__filename);
+
+async function generateSharingId(req, res) {
+  const newSharingId = uuidv4();
+  return await sqldb.queryZeroOrOneRowAsync(sql.update_sharing_id, {
+    sharing_id: newSharingId,
+    course_id: res.locals.course.id,
+  });
+}
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    debug('GET /');
```

Comment:
Do you actually find yourself using these debug prints? It's fine if you do, but otherwise, I'd rather remove them; I've found they don't add much long-term value for anything other than the most complex code.

Reply:
no I don't, I've removed them.